build/Version.props

```
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <TgsCoreVersion>6.19.0</TgsCoreVersion>
     <TgsConfigVersion>5.8.0</TgsConfigVersion>
-    <TgsRestVersion>10.14.0</TgsRestVersion>
+    <TgsRestVersion>10.15.0</TgsRestVersion>
```

```suggestion
    <TgsRestVersion>10.14.0</TgsRestVersion>
```
10.14 still hasn't released